learn/wavemaker-release-notes/v10-13-2.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "WaveMaker 10.13.2 - Release date: 11 April 2022"
+id: ""
+sidebar_label: "Release v10.13.2"
+---
+*WaveMaker released another version update with new features and bug fixes.*
+
+## Improvements
+---
+
+### File Upload
+
+**Clear** and **Delete** methods are introduced to clear and delete uploaded files. For more information, see [Clear Uploaded Files from the UI](/learn/app-development/widgets/form-widgets/file-upload#clear-uploaded-files-from-ui).
+
+### Textarea
+
+Introduced a new property called **Help text (character count)**, which shows when you set the **Maximum Characters** property of the [Textarea](/learn/app-development/widgets/form-widgets/textarea/) widget. You can choose a message from a provided formats list or create your own custom message, showing the number of allowed characters as you type. For more information, see the [Textarea widget Properties](/learn/app-development/widgets/form-widgets/textarea#properties).
+
+### Datatable
+
```

Comment:
Following issues have been fixed with Datatable widget when pagination is infinte scroll or on-demand scroll:

- You can retain the selected row details, including edited values, when scrolling to the extreme bottom for viewing more records.
- The expended rows now remain expended when more data is loaded on scrolling to the bottom of the table or clicking on the "Load more" button.